operator/src/main/java/oracle/kubernetes/operator/EventConstants.java

```diff
@@ -6,35 +6,34 @@
 /** Constants used in Operator events. */
 public interface EventConstants {
 
-  String DOMAIN_CHANGED_EVENT = "DomainChanged";
-  String DOMAIN_CREATED_EVENT = "DomainCreated";
-  String DOMAIN_DELETED_EVENT = "DomainDeleted";
-  String DOMAIN_PROCESSING_COMPLETED_EVENT = "DomainProcessingCompleted";
-  String DOMAIN_AVAILABLE_EVENT = "DomainAvailable";
-  String DOMAIN_COMPLETED_EVENT = "DomainCompleted";
-  String DOMAIN_PROCESSING_FAILED_EVENT = "DomainProcessingFailed";
-  String DOMAIN_PROCESSING_ABORTED_EVENT = "DomainProcessingAborted";
-  String DOMAIN_ROLL_COMPLETED_EVENT = "DomainRollCompleted";
-  String DOMAIN_ROLL_STARTING_EVENT = "DomainRollStarting";
-  String DOMAIN_VALIDATION_ERROR_EVENT = "DomainValidationError";
+  String DOMAIN_CHANGED_EVENT = "Changed";
+  String DOMAIN_CREATED_EVENT = "Created";
+  String DOMAIN_DELETED_EVENT = "Deleted";
+  String DOMAIN_AVAILABLE_EVENT = "Available";
+  String DOMAIN_COMPLETED_EVENT = "Completed";
+  String DOMAIN_FAILED_EVENT = "Failed";
+  String DOMAIN_ROLL_COMPLETED_EVENT = "RollCompleted";
+  String DOMAIN_ROLL_STARTING_EVENT = "RollStarting";
   String POD_CYCLE_STARTING_EVENT = "PodCycleStarting";
+  String DOMAIN_UNAVAILABLE_EVENT = "Unavailable";
+  String DOMAIN_INCOMPLETE_EVENT = "Incomplete";
+  String DOMAIN_FAILURE_RESOLVED_EVENT = "FailureResolved";
   String EVENT_NORMAL = "Normal";
   String EVENT_WARNING = "Warning";
   String WEBLOGIC_OPERATOR_COMPONENT = "weblogic.operator";
 
   String DOMAIN_AVAILABLE_PATTERN = "Domain %s became available";
-  String DOMAIN_CREATED_PATTERN = "Domain resource %s was created";
-  String DOMAIN_CHANGED_PATTERN = "Domain resource %s was changed";
+  String DOMAIN_CREATED_PATTERN = "Domain %s was created";
+  String DOMAIN_CHANGED_PATTERN = "Domain %s was changed";
   String DOMAIN_COMPLETED_PATTERN = "Domain %s is completely ready";
```

Comment:
`Domain %s is complete because all of the following are true: failed is not true, there are no pending server shutdowns, and all servers expected to be running are ready and at their target image, auxiliary images, restart version, and introspect version.`

Is that too long?

Reply:
How about:
Domain %s is complete because all of the following are true: no failure detected, no pending server shutdowns, and all servers expected to be running are ready and at their target image, auxiliary images, restart version, and introspect version.

Reply:
Sounds good to me. Your version is more clear.

Reply:
Similarly, please update the corresponding user doc.  (Ditto for the other "complete" and "available" rewordings).